packages/FormElement/src/components/Layout/index.js

```diff
@@ -0,0 +1 @@
+export { LayoutRightCol, LayoutLeftCol, LayoutFlexParent } from "./Layout";
```

Comment:
Instead of exporting each, what if we just export `Layout` as default?

Something like:

```js
const Layout = styled.div...
export const LeftCol = styled.div...
export const RightCol = styled.div...

Layout.LeftCol = LeftCol;
Layout.RightCol = RightCol;

export default Layout;
```

Then to use, it's like this:
```js
import FormElement from "../../src";
const { Layout } = FormElement;

<Layout>
  <Layout.LeftCol width="120">...</Layout.LeftCol>
  <Layout.RightCol>...</Layout.RightCol>
</Layout>
```

Does that seem cleaner to you?

Reply:
Yes but I was trying to keep consistent with the approach of the api, eg not having to use the `.` syntax. But I perhaps it's fine as they are all related subcomponents so perhaps in this case it does make it more clearer associating them in this way, will go with that